recipes/wtl/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, CMake, tools
+
+import os
+
+class WTLTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
```

Comment:
You should still add the `cmake` generator.
By using `conan_basic_setup`, this will modify the compiler settings appropriately.

Reply:
I prefer a non-intrusive approach - to not modify  CMake scripts and to use find_package for searching packages. This approach allows us to use Conan as package provider and replace it with another package manager or internal scripts if needed without interfering into CMake logic. But if `cmake` generator is recommended for CCI's test packages I will fix this, but still I haven't seen any guideline on this.

Reply:
You can combine more then one generator: the `cmake` generator will configure and check the current compile and `cmake_find_package` will handle the `wtl` dependency.

The part where it configure and checks the current compiler is important.
Also, it's nice to have all test_packages using the same concepts.

I opened https://github.com/vaerizk/conan-center-index/pull/1 to show you what I mean.

Reply:
By non-intrusive approach I meant that`conan_basic_setup` is not added to the original CMake scripts - in general case we may not want to modify them. But in case with conan test packages, since these ones are supposed to work only with conan, there is no need to consider other package providers and if other test packages prefer `cmake` generator, I agree, it is better to use it, and I think one generator is enough here atm.

Pull request merged in 4e18c23dcb8ba0e41f1b500c29a9692cda7e87cf
`cmake_find_package` generator removed in 85ef5923de04949385892f556549f9a4cc9fa5c8

Reply:
Yeah, I also try to keep the modification to a minimum.

Reply:
for this one, not that important, as it's header-only.